### DISCUSSION TITLE
New aliens: the Naltok.

### DISCUSSION THREAD

**Discussion by Quantumshark** [endless-sky/endless-sky#8737] [Ideas]
### Overview

The Naltok are a species located here, slotted in between the Kimek and the Rim but out of reach by JD from both, except in the southernmost tip of their space.
![image](https://github.com/endless-sky/endless-sky/assets/18634983/6aa6877d-5fea-49dd-bf8b-eb5cd1de4c6c)

The Naltok are intended to be roughly Tier 1.5, sitting somewhere between humanity and the Korath automata, marginally above the Hai, but with technology that specialises in efficiency rather than raw power.

<details>
<summary><h3>Society</h3></summary>
<br>
Currently, the Naltok are split between two main governments: the technocratic government of the Telashki and their Ministries, and the charismatic military leadership of the Xochira under the command of the Grand Admiral, Keloxok. There are also independent merchants, and a single neutral station observing the two white dwarfs in Vokegol. While the Telashki and Xochira have fought in the past, and are nominally still at war, they are not actually fighting each other - the only route between the two factions passes through Vokegol, but also the Xochira are sufficiently outmatched by the Telashki that they're reluctant to attack, while the Telashki have decided that having an enemy is in many ways quite useful.

The Telashki are led in a distributed system of government by the Ministries. There are several Ministries of various sizes, but the main ones are the Ministry of Transport, the Ministry of War, the Ministry of Alien Life and most of all the Ministry of Genetic Research. Each Ministry is primarily composed of experts in the relevant areas, and has direct power over only the decisions relevant to it.
In Telashki society, the population have been socially engineered over many years with the objective of streamlining the process of technological advancement. Secrecy of any form is taboo among the Telashki - civilians and the government alike are always open, and misleading or lying are likewise seen as attempts to obfuscate the truth.
Relatively recently, significant numbers of Alphas arrived in Naltok space, travelling via the wormhole. These Alphas, considered friends by the Telashki and enemies by the Xochira, are now trapped in Telashki space, unable to leave due to the vigilant patrols of the Xochira that cover all potential exits from Naltok space.

The Xochira are led by their military, and particularly revere their charismatic leader, the Grand Admiral, Keloxok. Most Xochira distrust the Alphas, the Telashki, or both, and despite their comparatively weak military position the Xochira's ultimate goal is to invade the Telashki and force the Alphas out of Naltok space. While at first glance the Xochira superficially resemble other militaristic factions like the Unfettered or the Vi, they do not share those factions' honour, nor do they place value in personal physical strength - rather, the Xochira strongly value tactical brilliance, not raw strength.
The Xochira subscribe to the Human side of the story about the Alphas, specifically to a single account of it from one human crew member the Alphas brought with them, an account the Telashki dismissed as misinformation and slander, and are repulsed by the fact that the Telashki treat these psychotic aliens better than their own citizens.
</details>

<details>
<summary><h3>History</h3></summary>
<br>
The ancestors of the Naltok were creatures similar to newts, evolved on Nareltok, an old, marshy planet orbiting a small orange K3 star. The notable parts of their history begin in their industrial period, where their physiology coupled with the fact that a majority of their fuel source came from peat rather than more efficient coals and oils made them particularly vulnerable to greenhouse warming. As their planet warmed and dried, their society collapsed into a massive global war, until they were approaching the brink of extinction.
Unable to repair their planet quickly enough to save themselves, the Naltok modified themselves into forms better suited to the self-inflicted harsh conditions of their planet - thick, waxy skin to reduce water loss, decreased size and slowed metabolisms to reduce their food requirements, and a dark radiation-resisting pigment to shield them from the effects of the nuclear fallout resulting from the war.
Having rescued their entire species from extinction, the group of scientists responsible for this initiative were able to take power, forming a single unified government. The new government blamed the disasters on the preceding governments' lack of scientific knowledge, and established their distributed technocratic system.
The end of the war, and the subsequent reengineering of the Naltok, ocurred around 5700 years before the start of the game.

With no vaguely habitable planets in their home system beside their moonless homeworld, and having identified the life-supporting moon Vigechox in their neighboring system, the Naltok sent out their first interstellar voyage disproportionately early relative to their technology, so the resulting colony was seprarated for around 300 years before the hyperdrive was developed. During this time, the lower gravity of the newly colonised moon, and the comparitive ease of transit to the other moons of the same gas giant, led to a much more developed shipbuilding industry than developed on their homeworld, where the local government heavily pushed for biotechnology over other advances, supported by the blind trust of their populace. Despite this, ultimately it was scientists on the homeworld that first invented the hyperdrive, reuniting the two societies. 

The reunification was peaceful and total, with free exchange of technology and free movement of citizens beginning almost immediately. Both sides experienced this as a major technological leap, and with the new Hyperdrives they quickly began expanding into the rest of their region of space. Soon after this, the Naltok encountered the Quarg for the first time, and after a brief delay excitedly initiated contact, expecting a peaceful unification similar to the unification of their two worlds that had occurred a few years earlier. The planet of Kutora, as close as possible to the Quarg ringworld, was settled and became home to the newly founded Ministry of Alien Life, and the Naltok began excitedly sharing their technology with the Quarg, in the assumption that the Quarg would reciprocate.

The Quarg, however, had no interest in granting their technology to the vastly less mature Naltok, particularly the Jump Drives which could allow the Naltok to meet the Coalition. The Naltok hopes of cooperation were slowly crushed into oblivion, and quiet hostility toward the Quarg began to grow throughout the population. The Naltok, at first hopeful, now abandoning their dreams of a peaceful cooperation and unification like that between Nareltok and Vigechox, began to attempt to decipher the actions of the Quarg. With great reluctance, the Ministry of Alien Life and Ministry of Genetic Research made a single exception to their policy of non-secrecy, out of fear of potential Quarg retaliation if discovered. Quarg genomes were sequenced in secret, using collected genetic material from their surroundings, and Quarg cloned, in an attempt to understand the physical neurology of the Quarg brain that ultimately yielded little of any use, and is generally viewed as a mistake by the few that are aware of it. Despite their resentment of the Quarg, the Naltok never considered that it might be possible to force them away, and so the Quarg were left to build their ringworld in peace.

(there is a gap in their history here that is several thousand years long that hasn't been written yet, but the main notable event here is their first contact with small Heliarch visits similar to those on Poisonwood)

Recent Naltok history is defined by the arrival of the Alphas. A single Alpha ship, with a Jump Drive dating back to a single Pug-enhanced Deep ship looted during the Alpha Wars, travelled from the upper edges of (former? need to check dates) CCOR space into Naltok space. The Naltok remained hopeful that the Quarg were the outliers in terms of interspecies relations - the exception, not the rule - and thus welcomed the Alphas with open arms, hoping to gain insights about any technology they might have. Though the Alphas could provide little of note, their technology being far behind that of the Naltok (and what they brought being somewhat outdated by human standards), the Alphas quickly realised that by playing along with the Naltok requests they could secure a base beyond humanity and perhaps begin manipulating and cheating the Naltok as they had sought to with the Hai. The Naltok quickly identified the hallmarks of genetic engineering in the Alphas, and were even more excited to discover a species with artificial genetics like their own. The Alpha ship travelled to human space and back several times, bringing across an entire enclave of Alphas and some of their Human subjects.

In fact, the Naltok government's treatment of its Alpha visitors was far better than that of its own civilians, and many resented this. In a series of brilliant speeches and tactical engagements, Keloxok rallied together a fleet on Xorechige, declared the independence of the Xochira (meaning "defenders") and captured the southern half of Naltok space before their opponents could react. The Naltok government rallied a fleet in Keloteg, preparing to jump to Lixetog, but as they took off they found the required hyperlane inexplicably absent, leaving no possible path to attack from. Not wanting to risk a battle in Vokegol, the newly christened Telashki (meaning "augmented", a description intended to include both the Naltok and the Alphas) relented, beginning the ceasefire which has continued to this day.
</details>

<details>
<summary><h3>Language</h3></summary>
<br>
The Naltok language constructs words from syllables, which are either consonant-vowel, or consonant-vowel-consonant. In practice, this means words (and, more to the point, names) in the Naltok language will always start with a consonant, and can't have consecutive vowels or more than two consecutive consonants (note, though, that some single consonant sounds are rendered with two letters in english text, such as th or ch).
They also have a more restrictive list of consonants than English: l k g ng v t s sh f j y ch x r th
While they don't have particular difficulty picking up words outside these restrictions when speaking the human language (which is mostly spoken by Naltok from the Ministry of Alien Life; other ministries will often work through a translator from the Ministry of Alien Life's Human Department), names for Naltok characters, locations and other such things are constructed using these rules, which will hopefully result in a consistent and distinct theme.
</details>

<details>
<summary><h3>Ships and Outfits</h3></summary>
<br>
Naltok ships have several distinctions from many other ships, thanks to their unique construction. While other species tend to build ships from metals or similar materials, Naltok ships are made of living bacterial colonies. This grants the Naltok a fairly powerful innate hull regeneration ability with very low energy requirements, at the cost of very poor heat dissipation and a permeable shield matrix as the bacterial hull grows over and partially blocks the shield projectors. This hull is also heavier than conventional materials, so Naltok ships tend to broadly have higher ratios of mass to outfit space than comparable ships of other factions.
Naltok ships designed before the Xochira-Telashki split are generally undermilitarised and make questionable design decisions, both factors resulting from the long Naltok history of peace. More recent designs by the two factions break this trend, being highly effective at the cost of civilian capabilities.

Naltok outfits are highly efficient at the cost of raw power.
Naltok shield generators have similar generation/ton to human shield generators, but use only 1/3 of the energy per shield point.
Naltok engines have similar thrust/ton and turn/ton to human plasma engines, but with extremely low energy requirements and heat generation that make them better than anything else currently in the game for thrust and turn per overall outfit space after cooling and energy - these are intended to provide a new niche in the engine sandbox, though I don't anticipate them being used anywhere near as often as Korath or Remnant ones, simply becasuse the metric by which Naltok engines are the best isn't the one most people optimise for. The Naltok also have larger turning than thrusters (though not nearly as much so as the Bunrodea), because with the high raw mass of their ships this gives their stock ships average top speed and turning but poor acceleration, and they don't have reverse thrusters.
Naltok reactors and generators have generally comparable energy/ton to human equivalents, but with massively reduced heat generation that pushes them up to T1.5 levels of overall efficacy. In addition to bioelectric cells, an RTG and a fission reactor, the Naltok also have biophotovoltaic cells, which provide half their energy as `solar collection` and half as `energy generation`.
The Naltok make significant use of biotechnolgy where possible, and this is an area that I hope to expand on during their campaign, particularly for the Telashki.

Naltok weapons generally have low energy requirements and heat generation, and are somewhat lacking in raw power for their size, but they can still pack a hefty punch in the right circumstances. Of particular note are their Shell-Splitter Torpedoes, massive staged missiles that use discarded armour plating to distract enemy anti-missile systems, and their Steel-Render, a large artillery-like turreted cannon.

In addition to their shared weapons, the Xochira have two weapons specifically designed to block hostiles from traversing their space without conflict. Halt-Swarm missiles are fast, but weak, and deal slowing damage, but the more notable of the two weapons is the Leech-Ray, a burst-firing laser which drains its targets' fuel reserves. Mounted on the impressively fast Archerfish, the Leech-Rays make Xochira space very difficult to travel through without preparation.
</details>

<details>
<summary><h3>Other Content</h3></summary>
<br>
The major non-Naltok feature of Naltok space is the Star-Worms of Norelon, a species of filamentous bioluminescent spacefaring bacteria, acquiring nutrients and energy from charged dust particles. Star-Worms propel themselves through space in a semicontrolled manner using tiny biological ion drives, and are noted to have a fondness for clinging harmlessly to ships' shields. These will feature in future Naltok content, but for now they simply serve as an example of less evolutionarily advanced spacelife.
</details>

Everything that I have done so far on the Naltok is available in plugin format here:
https://github.com/Quantumshark/ES-Naltok
I do not feel I am ready to PR the Naltok yet, as they are still lacking images for ships, outfits and visual effects, but I am very much open to suggestions and proposed changes to what exists currently, so if anyone wants to look in that repository or even PR changes to it I'd appreciate the input.

**Comment by roadrunner56** [comment]
This is very well thought out! I especially like the incorporation of Alphas into the lore, as I feel that they've been underutilized. I'm  excited about the development of this faction and hope to see it in-game soon.
One thing though, when did the Pug open that wormhole?

**Reply by Quantumshark** [reply]
The Pug opened the wormhole a little over 80,000 years before the start of the game, this would have been the wormhole the Wanderers used to go to their systems north of Hai space.

**Reply by DarcyManoel** [reply]
I may be remembering the details wrong but don't we have lore stating that the builder territory came directly before the old Hai space?

**Reply by Quantumshark** [reply]
I'm fairly sure that's not the case, on the grounds that iirc the Hai encountered the Builders during the large Hai empire period, while there were still Hai in the northern systems, and obviously the Wanderers must have been in Builder space before the Builders appeared, so I don't think it's possible that the Wanderers could have been in the Builder territory most recently before the northern Hai systems.

**Reply by Saugia** [reply]
@beccabunny

**Reply by beccabunny** [reply]
Quantum is correct, after moving away from the graveyard the Wanderers went to an unspecified territory before moving to the cluster north of hai space

**Comment by Arachi-Lover** [comment]
Only point of concern to me thus far is this `Quarg genomes were secretly taken, and Quarg cloned`, I'm not sure how well kept secret this could be, as it'd involve some manner of either stalking or hurting a Quarg in order to acquire the genetic material. Getting the material itself wouldn't be impossible, but keeping it a secret I feel would be very difficult, and if they Quarg did become aware, I imagine they wouldn't take well to having others try and clone them like that.
On the cloning itself, reading throug it feels like that kind of came a bit abruptly; how did they come to have this capability, and master it to the level they can achieve clones of alien species with just samples of the genetic material?

**Reply by Quantumshark** [reply]
Generally trace amounts of genetic material are shed into the environment somewhat by any species - it could be picked up from door handles and the like without needing to directly harvest it from the host. With a governmental level of planning and budgeting, there are probably quite a few ways to go about it without being noticed, unless that's something the Quarg were already on their guard about.
As far as the ability to clone them goes, the Naltok have a fairly good grasp of biotechnology, and present-day humanity isn't enormously far away from being able to do that, so it seems fairly plausible to me, but this would have been a project over a time period of decades, not something they could just do right off the bat - a long time for a research project, but a relatively short span of time on the scale of their whole history.

**Reply by Zireael07** [reply]
> Generally trace amounts of genetic material are shed into the environment somewhat by any species

Worth explicitly putting somewhere both in this writeup and in lore.

**Reply by Arachi-Lover** [reply]
Even still, the Quarg would have at least some manner of surveillance system in their stations and buildings, so at least the Naltok who got the material would have to deal with the Quarg asking what they were doing with such focus on door handles and whatnot, the suspiciously sneaky behavior at least, if not directly confronting about the cloning.

**Reply by Quantumshark** [reply]
The Naltok would have taken steps to conceal the actual action - I'm imagining possibly something either concealed up a long sleeve or more likely built into the cuff of the sleeve itself, something that could easily be 'inadvertently' brushed against any piece of furniture without drawing attention. Even one single sample taken on one occasion would, with a sufficiently precise shotgun sequencing approach, probably yield useable full genomes of multiple Quarg, from which they could then start working on the cloning part.

**Comment by roadrunner56** [comment]
In regards to the lack of sprites for ships and outfits, you could always put a request for art in the Discord if you haven't already.

**Reply by Saugia** [reply]
I may or may not have attempted one already to Doom's liking.

**Reply by Saugia** [reply]
![naltok](https://github.com/endless-sky/endless-sky/assets/93169396/8dbc5608-efa1-46ea-a578-a466550fa520)

This is a remodel of the small freighter. I would be interested in making models for all of the ships. My only concern is having dark and light greys again, which is associated with the Quarg, and a few other planned species also want a similar palette. I'd prefer to have them tinted towards colors, or have more colors showing.

**Reply by Quantumshark** [reply]
If you're interested in making models for all the ships, that would be incredibly helpful, and I would really appreciate it.

As far as colours go, my original ideas avoided anything beyond the standard "grey and red/orange/yellow so the swizzle isn't an issue", in part because I'm partially colourblind, so the necessary careful observation of how the colours look at each swizzle isn't something I can actually do. There is a requirement for different swizzles on the ships to be clearly distinct, because the ship lineups of the different Naltok factions have a lot of overlap, so in a lot of cases you'll have the same ships used by multiple factions and thus only be able to visually distinguish them by swizzle. 
In short, I'd be OK with giving them a different colour scheme, but wouldn't be able to provide much in the way of useful comment on it.

**Comment by Fotiadis-110** [comment]
I've recently been fascinated by the thought of a bulk hauler ship. A big empty box of a ship with probably 100 crew required.
And with 1000 empty outfit space to drop in cargo or crew modules.
2000 space would mean 100 of each, 400 crew, 1500 cargo hauling. So 1000 available volume would be 200 bonus crew and 750 cargo hauling.

Such a ship would need next to no cargo space. Could choose to become a magnificent tank adsorbing damage. So presumably it would be important to not over protect, nor over arm the craft, it's a cavernous hulk to be customized as you desire, not a true warship. And having significant spare engine space would tempt you into improving mobility, at the cost of consuming some of the available versatility.

**Reply by Fzzr** [reply]
While up to a point customizability is a good thing, we want to avoid best-for-everything ships. The Bactrian is currently the most modifiable ship in the game, and it's also our single biggest game balance headache. This would be a Bactrian++ and essentially invalidate the vast majority of other ships. In particular, being able to get a very high crew count could make it the best ship in the game for capturing, which would further exacerbate our other biggest game design headache, the dominance of capturing as a way to advance in power without interacting with the story at all.

Instead, we're adding customizability by giving each species different ways to optimize, like the Korath combining shield and battery, the Wanderers combining engines and cooling, etc. This allows the player to build ships that are good at different play styles while putting interesting constraints in place to avoid an objectively best choice for anything.

**Reply by Fotiadis-110** [reply]
Actually. I was thinking give the ship next to no cargo capacity. maybe 10. for 'war trophies'
And with 100 base crew, and 1000 outfit space the maximum crew goes to 300. inferior to the Kirmek Spire at 350 crew, or the Bactrians max of 400 and change. The Spire is my preferred Capture boat because of better weight/mobility stats, lower required crew to land on planets and less eternal overheating problems in the Wastes.
We do of course have to be cautious of making too much outfit space overall available, otherwise all other ships become obsolete, but at the same time, a big empty brick of a ship is an interesting diversion from the standard preset hull designs, Just how many passangers do you want?

As it stands this race in particular will not benefit much from the cavernous hull, but the Capture Bactrian which suffers from punishing thermal problems will get: Cooler reactors, Cooler thrusters, and energy efficient shields. So this race as a whole buffs capture Bactrian to be better at capturing. To balance this it could be hilarious to make their reactor technology reduce thermal efficency, punishing the Bactrian for being cheeky.

So on the whole, my Multipurpose Ship can choose its Crew/Cargo/Shield/Repair blend to become extra tough, or a wannabe Spire, or a wannabe Geocoris. It lacks the talent to be as 'overpowered' as the Bactrian or Dromedary which both cost a fortune and are supposed to be top grade ships, Arguably the only 'fix' might be locking Bactrian behind the Free Worlds (as Hero of the Pug invasion, we believe you might be worthy of purchasing these rare ships, usually only available to Citizens of the Deep, and give us a 'lesser reward' for the deep unlock quest chain a free Mule?) and unlock it for the Hai. 

My concept is supposed to just be interesting and different in style to 90% of ships, and probably built as a civilian tug, thus be lacking in weapons slots. Offering creative and flexible or roleplay choices to your purchase.

Arguably, we can fix the Bactrian by cutting it's default stats and restricting cargo Cap. Heck lowering its crew needs and bunk availability by 25 and 50 would change its meta playstyle immediately.

**Reply by Saugia** [reply]
Is this something you're suggesting for the Naltok specifically? If so, mention them and how you think this type of ship would fit for them. Though Doom has shown that he does not believe this type of ship would be fitting for the Naltok.

If this is an idea you had for a ship, or something you'd like to see in general, please move it to its own discussion. This discussion is specifically for the Naltok's lore and creation.

**Reply by Fotiadis-110** [reply]
At present, we have plenty of human ships, and Hai, and Korath. Even Coalition.

So to fit the philosophy of a race you would need creative thinkers, or a race pressed to need to adapt and overcome. Like the Wanderers (who have dedicated transport and Cargo haulers, of which which they already had to press into service as passenger tugs in the past), or perhaps a race under pressure from within. Like Technocrats in a Civil war. Oh hey? Look what popped up! Now technically the Remnant could do something similar, flexible ships are something they enjoy, but they mostly appreciate carefully crafted purpose built designs as a whole.

I'm not trying to demand it be added to the Naltok, merely offering a suggestion to operate as both the Liner and Heavy Cargo roles with a side of showing my working for balance purposes, to take one model and Lore and make it do triple duty as the Distraction Carnifex, Hauler, and Cargo ship. Which would be a fun surprise to anyone who wanders into their space in hostile roles.

Why. Won't. You. DIE? is always a fun reaction to the player who doesn't bother carrying their scanning, and even then: 'wait. How many shield generators?!?'

**Reply by Quantumshark** [reply]
This ship concept isn't really one the Naltok would be likely to produce. The Ministry of Transport, which is both the manufacturer and the operator of the majority of the Naltok civilian ships, has no shortage of funding, and is working with a fairly consistent averaged-out demand for both passenger and cargo transport across Naltok space, so they can afford to benefit from simultaneously operating specialised passenger liners (the Marlin) and specialised cargo ships (primarily the Conger, with the Flounder used for transporting single large items), and wouldn't find it useful to be able to convert between the two.
So, the idea of a single highly modular ship which can potentially fill either role at a given time isn't something that really gels with the situation Naltok ship designers are in.

**Comment by TheGiraffe3** [comment]
Perhaps the Naltok speech pattern could have a lot of the repeated letter s? (e.g. sssalamander and sssnake)
That's how a lot of books in which amphibians can talk make them speak.

**Reply by Quantumshark** [reply]
I do have fairly developed ideas about the phonotactics of the Naltok language (incidentally, I could probably mention that in this discussion somewhere) - that's what I've been using to name their systems, factions, and characters. As is, I think, visible from their system names, their language doesn't particularly lean towards a greater number of sibilant sounds (conversely, I believe MZ mentioned in passing somewhere that the Korath, being lizards, have a language that _does_ lean in that direction), so I'm not sure it quite makes sense for them to speak in such a manner. There's no reason it couldn't work for another species though, and I don't think it's a bad idea.

One of the earlier ideas I had was to try to give them a 'phonetic accent', restricting them to using only sounds that could exist in the Naltok language and trying to render english words within those restrictions. The results were certainly distinctive, but they were also pretty much impossible to read, even with my advantage of having written them and hence known what was supposed to be being said.

Beyond that, though, if possible I'd like to construct their speech in a manner that reveals something about their culture, about the way they think, and how that's different from what's typical for humans, rather than a manner that's connected to their physiology. I haven't yet managed to do that in a way I'm happy with, but that is my goal.

**Comment by Fotiadis-110** [comment]
Oh I used both, had the problem. Shifted a system 5 up and 5 right after a
lot of searching. Or was it .5? It's been a while.

Solved the problem without significantly altering the world map.

I actually wondered why the mod manager didn't pick up the issue, till I
concluded it would make the whole 'alternate map' in the forgotten content
non-viable.
